net-irc/irssi/patches/irssi-0.8.17-haiku.patch

```diff
@@ -0,0 +1,315 @@
+From e9617d64b8b8846dd7c7379cab59fe6ee85ca629 Mon Sep 17 00:00:00 2001
+From: pisculichi <pisculichi@openmailbox.org>
+Date: Fri, 18 Dec 2015 00:12:26 +0000
```

Comment:
Btw, don't you know git am ? it applies patches as commits with keeping the author :p